src/validators/ApplicationValidator.ts

```diff
@@ -0,0 +1,17 @@
+import { Application } from "../models/application";
+
+export const validateApplicationObject = (application : Application) => {
```

Comment:
Every field is being internally generated and therefore you do not need to validate as this is not user input.

Reply:
Deleted now